### DISCUSSION TITLE
Skip checksum creation

### DISCUSSION THREAD

**Discussion by ViRb3** [goreleaser/goreleaser#1623] [Ideas, Issue Triage, and General discussions]
Hello,

I am building a project that requires a separate Windows builder and Linux builder. When the first one uploads the artifacts, everything is OK, but when the second one attempts to upload the checksum file, it obviously already exists, so it errors out. Is there a way to fix this, e.g. skip the checksum creation?

Thanks

**Comment by ViRb3** [comment]
Just an update, I worked this around by setting different names for the checksum file. It would still be nice to be able to disable this feature completely though.

**Reply by caarlos0** [reply]
seems like a good workaround, thanks for sharing!

**Comment by caarlos0** [comment]
For now there is no way to disable it, but I created https://github.com/goreleaser/goreleaser/issues/1628 to handle it.
